aspnet/dnx/projects.rst

```diff
@@ -1,8 +1,652 @@
-.. include:: /../common/stub-topic.txt
+Working with DNX Projects
+=========================
 
-|stub-icon| Working with DNX Projects
-=====================================
+DNX projects are used to build and run .NET applications for Windows, Mac and Linux. DNX provides a host process, CLR hosting logic and managed entry point discovery. 
 
-.. include:: /../common/stub-notice.txt
+.. note:: To see and create an example of a DNX project, follow the steps provided in the tutorial `Create an ASP.NET 5 web app in Visual Studio Code <https://azure.microsoft.com/en-us/documentation/articles/web-sites-create-web-app-using-vscode/>`_. After creating the app and opening it in VSCode, you will be able to view the **project.json** file.
 
-.. _issue: https://github.com/aspnet/Docs/issues/50
\ No newline at end of file
+Creating a new project 
+----------------------
+
+At its simplest form, a DNX project is a **project.json** file along with a code file:
+
+::
+
+    -MyApp (folder)
+    --project.json
+    --Program.cs
+
+There are two mandatory conditions for a project. The *project.json* file must contain valid json, where brackets *{ }* are used inside the file, and that your *program.cs* file contains valid C#. 
+
+The presence of a *project.json* file defines a DNX project. It is the *project.json* file that contains all the information that the DNX needs to run and package your project. For additional details, including the *project.json* file schema, see `Project.json <https://github.com/aspnet/Home/wiki/Project.json-file>`_.
+
+When using some editors there are other files that you'll see. For instance, a file with the extension .xproj is used in the case of Visual Studio. These other types of files are requirements of their specific tool, but are not a requirement of the DNX. The .xproj file type, for example, is an MSBuild file that is used by Visual Studio and keeps information that is important to Visual Studio, but this type of file does not impact the DNX.
+
+Targeting frameworks
+--------------------
+
+One of the sections you can add to the *project.json* is the ``frameworks`` section. The ``frameworks`` section looks like this:
+
+.. code-block:: json
+
+    {
+      "frameworks": {
+        "dnx451": {},
+        "dnxcore50": {}
+      }
+    }
+
+Each of the values in the ``frameworks`` section of the *project.json* file is a framework that your project will compile. The above snippet will build for the full .NET Framework (dnx451) or Core CLR (dnxcore50). 
+
+If you pack this project, using ``dnu pack`` then you will get a NuGet package that looks like the following:
+
+.. image:: projects/_static/TargetPlatformNupkg.png
+
+Notice that the ``nupkg`` has a folder for each of the frameworks you specified, allowing this NuGet package to work on any of the frameworks. You can also specify this as follows:
```

Comment:
Suggestion: "You can also specify a .NET Portable profile by using the full name of that provide, for example:"